### DISCUSSION TITLE
👩‍🌾 Sunflower Fashion Week #4

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2828] [Art] [answered]
Welcome to the 4th Bumpkin Competition a.k.a Sunflower Fashion Week! 💃

The Art Team will choose the best outfits and announce the winners on the first week of every month! 😍

There's no specific theme for this competition! Designers are encouraged to use their imagination and come up with original ideas. 

Before continuing, it is recommended to read the Bumpkin Guideline, which can be found here:

https://docs.sunflower-land.com/contributing/for-designers/bumpkin-guidelines

## Requirements ✏️

- Dimensions of file should be 100x100px.
- The layer (e.g. shirt) should be on it's own layer without any other parts. When placed on top of a Bumpkin it should perfectly align with its body.
- Art should be original and not plagiarised.
- If submitting multiple items, an aseprite file would be appreciated to speed up review. 🤗
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely. Players may trade your item without the need to pay you royalties. The community may make changes and improve artwork that is submitted.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/bumpkin-art-competition#terms-and-conditions) of the game.
- Bumpkins live inside of the game! Please include an NPC version of your design that can be used inside of Sunflower Land.

When submitting art, please include the following:

- Your farm ID.
- A minified version of your art piece.
- An example of the art used inside of the game or on the Bumpkin. Here's a nice submission example from a community artist: https://github.com/sunflower-land/sunflower-land/discussions/2639#discussioncomment-6238691

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however this competition is not just about the prize money. It's an opportunity to showcase your incredible talent and become an essential part of the Sunflower Land experience. 

We want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Designers have until **September 4th** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of September.

We can't wait to see what amazing designs you come up with. 

Good luck to all participants! 🌻

Last Bumpkin Competition: #2639

**Comment by LTNGx39** [comment]
### <h1> Mystery visitor / Noir suit
**Someone unknown arrive to plaza**

`Farm ID: 161769`
`Discord: ncnlx39`

**Preview:**
![Noir_x5_png](https://github.com/sunflower-land/sunflower-land/assets/131133076/d752a699-b835-459c-8831-048b97f69468)

| Skin | Hat | Hat mask | Hair | Coat | Shirt | Pants | Shoes |
| - | - | - | - | - | - | - | - |
| ![noir_skin](https://github.com/sunflower-land/sunflower-land/assets/131133076/76d2c4ae-df6b-470b-81e0-3a8760281c08) | ![noir_hat](https://github.com/sunflower-land/sunflower-land/assets/131133076/f957c59f-7810-4d38-9cc9-ed4765c9489d) | ![noir_hat_mask](https://github.com/sunflower-land/sunflower-land/assets/131133076/0befe690-c8db-466f-9721-b7d93bea6852) | ![noir_hair](https://github.com/sunflower-land/sunflower-land/assets/131133076/96ba254c-b1c0-4b75-ba23-9b2d9bcac348) | ![noir_coat](https://github.com/sunflower-land/sunflower-land/assets/131133076/7c185264-5800-497c-b5c7-b1076481024a) | ![noir_shirt](https://github.com/sunflower-land/sunflower-land/assets/131133076/37ba2664-b92f-49e5-ba27-538707ec40c0) | ![noir_pants](https://github.com/sunflower-land/sunflower-land/assets/131133076/44aeedfd-b335-47bc-82f1-9a6897b4e286) | ![noir_shoes](https://github.com/sunflower-land/sunflower-land/assets/131133076/9e252efa-5fdc-453c-944a-c134fbf1546c) |

**Chibi version:**

![Chibi_x10](https://github.com/sunflower-land/sunflower-land/assets/131133076/8e054b2f-6bd8-48b4-af1c-e978330965a9)

| GIF | Sprite sheet |
| - | - |
| ![Chibi](https://github.com/sunflower-land/sunflower-land/assets/131133076/5a8668f6-5af6-42ae-80dd-9baddbf31e19) | ![spritesheet](https://github.com/sunflower-land/sunflower-land/assets/131133076/ecf202fd-610e-41c0-ad74-322f2dfd2cbb) |

**Proposals for use:**

 **1. Random appearances in the plaza**
> In the absence of an npc that exchanges items for sfl, they could add the mysterious visitor and similar to wanderleaf, he appears in a random place after 0:00 UTC, to promote the exploration of the players through the plaza, to worth it, he could ask for anything (unlike other npc who are limited in resources or food for example) and here pay a little more than usual (without being too clear, maximum 50% more)
To make it more immersive, his name does not appear on it, so it is not so easy to find

![propuesta](https://github.com/sunflower-land/sunflower-land/assets/131133076/3c0c8b6c-f64d-4c10-a606-732c1aa2190c)

 **2. Main NPC for the Halloween event or some event after**
>  I leave this for you to consider

**3. Full set with stylist**
>  I would love to be able to wear these clothes throughout the plaza

**Reply by VZozul** [reply]
Wow! Very cool. The only thing missing is something in the hand.. you still have time.. maybe a crow sitting on your hand.. or a lamp like "Dawn Lamp"

**Reply by LTNGx39** [reply]
I will take your comment into account, I will see if I add something else, I was also thinking of a background but it did not turn out as expected

**Comment by peperol** [comment]
what happens during the night in the enchanted woods?..
goblin faery hunter
![goblin faery hunter](https://github.com/sunflower-land/sunflower-land/assets/114354077/f8a30f9f-50ca-44ac-9bf6-ed0de956b42a)
human faery hunter
![human faery hunter](https://github.com/sunflower-land/sunflower-land/assets/114354077/fb1e557b-2cd7-4def-9e7e-d6af49c193b6)
enchanted Forest Background
![foresta incantata](https://github.com/sunflower-land/sunflower-land/assets/114354077/92d39773-63ef-4340-9921-117f2ba195c1)

hunter shirt
![camicia](https://github.com/sunflower-land/sunflower-land/assets/114354077/47dbe3cb-6760-409d-ab94-3cb59cd0ceaf)
hunter pants with suspenders
![suspender pants](https://github.com/sunflower-land/sunflower-land/assets/114354077/fecb7413-491d-4692-be5e-30df1782b3e8)
hunter boots
![shoes steampunk](https://github.com/sunflower-land/sunflower-land/assets/114354077/0fa96795-eb74-4f98-b445-44f6ebdce401)
hunter steampunk hat
![hat ](https://github.com/sunflower-land/sunflower-land/assets/114354077/9eba24a9-1437-4875-94f1-fadc95f27783)
goblin faery cage
![faery goblin](https://github.com/sunflower-land/sunflower-land/assets/114354077/b77a2a0a-c08c-4244-a9ad-48938c17caa0)
human faery cage
![faery human](https://github.com/sunflower-land/sunflower-land/assets/114354077/25f57ada-b613-4cf5-9da1-3d7f8b8f242a)
NPC
![steampunk goblin](https://github.com/sunflower-land/sunflower-land/assets/114354077/c10e8469-95a9-4b22-be46-23c5c08d29a9)
![steampunk](https://github.com/sunflower-land/sunflower-land/assets/114354077/5b852ea5-0d0e-414c-8073-e9bfcb9fb96d)
in land
![in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/7260f7d8-606d-42f3-be98-dec617bd6e3c)

Do you want a cup of moka coffee from your new neighbor Nanny Dimma?...;))

![nonnina back](https://github.com/sunflower-land/sunflower-land/assets/114354077/008aabf9-db22-4b43-8e95-f757f1ba9a19)

granny outfit 
![outfit](https://github.com/sunflower-land/sunflower-land/assets/114354077/7ab17b08-5a39-4879-9e9b-525a08775eb7)
wool shawl
![scialle](https://github.com/sunflower-land/sunflower-land/assets/114354077/aa624072-9473-4212-9bbb-53a605e581fa)
granny hair with curler and hat
![capelli](https://github.com/sunflower-land/sunflower-land/assets/114354077/f64302d8-6022-4058-8c19-9ea9476f6d51)
granny sandals 
![sandali](https://github.com/sunflower-land/sunflower-land/assets/114354077/2e7a056b-bec4-408e-b24b-29b50c4f8546)
glasses
![occhiali](https://github.com/sunflower-land/sunflower-land/assets/114354077/e3c3b81b-a1e8-4358-9463-3e630b70f373)
coffee maker
![caffettiera](https://github.com/sunflower-land/sunflower-land/assets/114354077/cbbe4713-6810-4f39-ba54-dfe0cb78d617)
the inseparable cat
![gatto](https://github.com/sunflower-land/sunflower-land/assets/114354077/8c62903e-b4f0-4ae7-82dd-b9952a0d0ccc)
granny skin
![skin nonna](https://github.com/sunflower-land/sunflower-land/assets/114354077/14628767-fb61-4635-9b5f-e938156204d8)
NPC in land
![Nonna in land](https://github.com/sunflower-land/sunflower-land/assets/114354077/26a02204-2bf9-4f76-934d-4d2bfb487933)
NPC
![Nonninaok '](https://github.com/sunflower-land/sunflower-land/assets/114354077/507d190a-c6dd-458f-a367-8e1c1bd1ea1a)




farm ID 145849
discord  peperol

**Comment by IceMonky29** [comment]
### **> Raven Style**


![Bumpikn Fashion](https://github.com/sunflower-land/sunflower-land/assets/142521805/bde18b99-ce45-449e-951a-2afb20454e3d)


| Skin | Hair | Rare Hat | Raven Shirt |
| - | - | - | - |
|![Body](https://github.com/sunflower-land/sunflower-land/assets/142521805/604e949c-3307-486d-acd9-2f72090b59de)| ![Hair](https://github.com/sunflower-land/sunflower-land/assets/142521805/5ad95c52-2020-470b-a690-7d5f918f99a2)|![Rare Hat](https://github.com/sunflower-land/sunflower-land/assets/142521805/84741f3c-18c9-46ba-858d-55c74773ed8b)|![Raven Shirt](https://github.com/sunflower-land/sunflower-land/assets/142521805/0c5f1e76-70af-4acf-adf4-f5591240a69a)

| Raven Pants | Shoes | Fondo |
| - | - | - |
|![Raven Pants](https://github.com/sunflower-land/sunflower-land/assets/142521805/31413bf9-a474-44a9-bf33-1c956dc7d2a3)|![Shoes](https://github.com/sunflower-land/sunflower-land/assets/142521805/cb04596e-3134-470b-b4fe-9fb6e57efa2e)|![Fondo](https://github.com/sunflower-land/sunflower-land/assets/142521805/32f84d66-e13d-49d2-ba4c-2d26ea4a216d)

| GIF | Sprite Sheet | 
| - | - | 
|![idle NPC](https://github.com/sunflower-land/sunflower-land/assets/142521805/97ff2969-f720-4c07-8cdc-790981cbf264)|![NPC Animacion](https://github.com/sunflower-land/sunflower-land/assets/142521805/57042f24-48a2-425d-993a-f49493b2e63d)

![idle NPC Isle](https://github.com/sunflower-land/sunflower-land/assets/142521805/978613cd-70c9-433d-a629-15ffcec1dcc0)

Fram ID: 162510
Discord: icemonky

In the background I added some of the hat designs that I prepared for the buds, I hope you like my style

**Comment by mmaxxpeinn** [comment]
# Jester Bumpkin # 
![jester](https://github.com/sunflower-land/sunflower-land/assets/120295844/79cd8bcf-d442-458a-b0b1-f0a73046f326)

| Hat | Suit | Pants | Shoes | Staff |
|-----|------|-------|-------|-------|
|![jester_hat](https://github.com/sunflower-land/sunflower-land/assets/120295844/6a1b4651-aa7f-443f-92d2-2c8d4bd1e2dc)|![jester_dress](https://github.com/sunflower-land/sunflower-land/assets/120295844/74526b4e-8f13-4b6b-aad1-7db95f1699db)|![jester_pants](https://github.com/sunflower-land/sunflower-land/assets/120295844/5154ad99-12b3-4ebf-be3f-a7dea22fdf48)|![jester_shoes](https://github.com/sunflower-land/sunflower-land/assets/120295844/7972e44f-b614-4def-99e9-1100470bb42e)|![jester_staff](https://github.com/sunflower-land/sunflower-land/assets/120295844/1026b886-185a-4a3a-818e-f63e2c750447)|

**Jester Bumpkin NPC**
![jester_npc](https://github.com/sunflower-land/sunflower-land/assets/120295844/f96cd702-bf52-4131-b617-32e855d4683a)

| Hat |![jester_hat_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/24a1ba59-90ec-489e-9e9e-3ed517c7e84b)|
|-----|-------|
| Suit |![jester_dress_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/f4e61d76-9a7c-4192-b02c-d340caee0553)|
| Pants |![jester_pants_sheet](https://github.com/sunflower-land/sunflower-land/assets/120295844/c78d8ad6-d9ab-494e-954d-14bfbdf4a1c4)|

Farm ID: `82122`
Discord: `mmaxxpeinn`

**Comment by andand0** [comment, marked as answer]
### Tourist outfit
<hr>


`Farm ID: 165810`
`Discord: Andando `


![tourist-style-500](https://github.com/sunflower-land/sunflower-land/assets/142132586/414c1826-ce38-4db5-a5f4-2897352a2ec6)

_This passionate traveler roams the world in pursuit of new culinary experiences. Exploring markets and local eateries to savor authentic dishes, delving into the stories and traditions behind each bite._

<br>

| Hat| Masked hat| Shirt| Pants|Shoes|Selfie stick|
|--------|--------|--------|--------|--------|--------|
|![tourist-hat](https://github.com/sunflower-land/sunflower-land/assets/142132586/805a2b60-c95c-49d7-973e-42b7957b9d40)|![tourist-masked-hat](https://github.com/sunflower-land/sunflower-land/assets/142132586/ec20046c-8996-43f4-ad38-c11fa6b79544)|![tourist-shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/8180c6f3-6939-4db1-88e7-990d404d4a01)|![tourist-pants](https://github.com/sunflower-land/sunflower-land/assets/142132586/ecb0c96b-890b-4540-bbae-32f6350976ca)|![tourist-shoes](https://github.com/sunflower-land/sunflower-land/assets/142132586/2f8f040c-df25-4f4e-84e6-315b8fe53644)|![tourist-selfie-stick](https://github.com/sunflower-land/sunflower-land/assets/142132586/1a6b8fca-51c7-45fb-b0a1-815a9f18a76f)|

<hr>

### NPC version 

![tourist-NPC-preview](https://github.com/sunflower-land/sunflower-land/assets/142132586/a8808d7d-b065-427a-9592-115775be61d8)

<hr>

| Gif | Sprite|
|--------|--------|
|![tourist-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/d375d767-ba54-4bca-82e4-9b62afe74c98)| ![tourist-NPC-sprite](https://github.com/sunflower-land/sunflower-land/assets/142132586/f7196ad6-256f-403e-b71a-5b8d31090d44)| 

| Sprite hat| Sprite shirt| Sprite pants|  Sprite shoes |
|--------|--------|--------|--------|
|![tourist-NPC-sprite-hat](https://github.com/sunflower-land/sunflower-land/assets/142132586/d38f7cf8-aff3-4d5a-8ec7-bfd472a106f1) | ![tourist-NPC-sprite-shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/57c571bc-5bbd-400b-ab46-908de49c2653)  |  ![surfist-NPC-sprite-pants](https://github.com/sunflower-land/sunflower-land/assets/142132586/5b043d36-741b-434c-8ef8-93f41a13d5de) |  ![tourist-NPC-sprite-shoes](https://github.com/sunflower-land/sunflower-land/assets/142132586/ba0718b3-91cd-4f9a-82c6-45c2a44147c1)| 

 <br> <br> <hr> <hr>  <br> <br>

### Surfer Style
<hr> <br>  <br>


![surfist5](https://github.com/sunflower-land/sunflower-land/assets/142132586/3ca4f0eb-6a11-4bce-9b7d-ae6a215badb0)

| Skin| Hair| Shirt| Pants| Shoes| Board |
|--------|--------|--------|--------|--------|--------|
| ![surfer-skin](https://github.com/sunflower-land/sunflower-land/assets/142132586/8f20a151-64f5-43fa-a819-e0b0064ba320) | ![surfer-hair](https://github.com/sunflower-land/sunflower-land/assets/142132586/680b30ce-4ef8-4a4b-a9cb-8cf30a621c81) | ![surfer-shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/56c48657-d350-4c46-91ca-fd356bfa2be0)| ![surfer-shorts](https://github.com/sunflower-land/sunflower-land/assets/142132586/24936403-8a30-4bc0-94a3-c150d99bec34) |![surfer-sandals](https://github.com/sunflower-land/sunflower-land/assets/142132586/7c210e2e-9d6c-454e-9a17-b97d0cac3180) | ![surfer-board](https://github.com/sunflower-land/sunflower-land/assets/142132586/083f71e8-e6eb-4c05-837a-a1b8b7468d7e)| 

<hr>

### NPC version 

<br>

![surfist-NPC-preview](https://github.com/sunflower-land/sunflower-land/assets/142132586/ce8a5bd4-7987-4694-986f-2462334f464c)

| Gif | Sprite|
|--------|--------|
|![surfist-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/3fe62f96-4458-47c4-9e79-53b94ac9a8de)| ![surfist-NPC-sprite](https://github.com/sunflower-land/sunflower-land/assets/142132586/f423b159-612e-4274-a109-75d613bb00f8)|

|Sprite hair|Sprite shirt|Sprite pants| 
|--------|--------|--------|
|![surfist-NPC-sprite-hair](https://github.com/sunflower-land/sunflower-land/assets/142132586/24703732-5829-400c-9e95-dbab56338fad)| ![surfist-NPC-sprite-shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/6595e227-aa88-40d8-bf72-06a5430fa080) | ![surfist-NPC-sprite-pants](https://github.com/sunflower-land/sunflower-land/assets/142132586/08a22447-6706-4bfa-9d58-3ab8da5b65e6)|

**Reply by LTNGx39** [reply]
bro this outfit looks great

**Reply by andand0** [reply]
Thanks. In fact, after upload it I'm wondering if I should make a backpack too but I'm not sure.

**Comment by ITsMeRBO2** [comment]
Hope U Enjoy The Idea ❤️  

![wen](https://github.com/sunflower-land/sunflower-land/assets/125754949/3709d81e-e1fc-4083-b42b-c2ab1bc500f3)


Without BackGround 👍 
![last du](https://github.com/sunflower-land/sunflower-land/assets/125754949/3f49dc17-6093-4df6-90e3-2d7f5cee68f3)


NPC : 
![giiif-export](https://github.com/sunflower-land/sunflower-land/assets/125754949/184f8bb7-eb54-49f8-bf0d-4381530744d2)



💭  Head : 
![HEAD](https://github.com/sunflower-land/sunflower-land/assets/125754949/2d535626-34ed-4932-a392-675f7e02250b)

💭 Body :  
![body](https://github.com/sunflower-land/sunflower-land/assets/125754949/9c2721ac-61a7-4fc8-9c25-624111baf891)

💭 Shield : 
![shild](https://github.com/sunflower-land/sunflower-land/assets/125754949/4c3fcd51-0b75-45ca-a2f6-9b7234ed637b)

💭  Sword : 
![sword](https://github.com/sunflower-land/sunflower-land/assets/125754949/b9159bf5-ce75-4882-aa2b-c7960d7078d6)




Farm ID : 143993
Discord : itsmerb.
